{
  "success": false,
  "return_code": 1,
  "stdout": "\ud83c\udf19 Moon Dev: Initialized FibonacciCrossover Strategy \u2728\n\ud83c\udf19 Moon Dev: LONG Entry at 16722.83 | SL: 16695.604973901332 | TP: 16777.280052197344 | Size: 59 \ud83d\ude80 Harmonic Confluence Confirmed \u2728\n\ud83c\udf19 Moon Dev: Closed Long on Bearish Crossover at 2023-01-03 11:30:00 \ud83d\udcc9\n\ud83c\udf19 Moon Dev: LONG Entry at 16738.86 | SL: 16712.793356431757 | TP: 16790.993287136484 | Size: 59 \ud83d\ude80 Harmonic Confluence Confirmed \u2728\n\ud83c\udf19 Moon Dev: LONG Entry at 16847.96 | SL: 16805.259654166424 | TP: 16933.360691667145 | Size: 58 \ud83d\ude80 Harmonic Confluence Confirmed \u2728\n\ud83c\udf19 Moon Dev: LONG Entry at 16833.01 | SL: 16796.393856812814 | TP: 16906.242286374367 | Size: 59 \ud83d\ude80 Harmonic Confluence Confirmed \u2728\n\ud83c\udf19 Moon Dev: Closed Long on Bearish Crossover at 2023-01-05 22:45:00 \ud83d\udcc9\n\ud83c\udf19 Moon Dev: LONG Entry at 16853.02 | SL: 16826.656894593958 | TP: 16905.74621081209 | Size: 59 \ud83d\ude80 Harmonic Confluence Confirmed \u2728\n\ud83c\udf19 Moon Dev: SHORT Entry at 16721.35 | SL: 16772.431490307332 | TP: 16619.18701938533 | Size: 59 \ud83d\udcc9 Harmonic Confluence Confirmed \u2728\n\n",
  "stderr": "Traceback (most recent call last):\n  File \"/Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_pp/10_23_2025/backtests_final/T03_FibonacciCrossover_BTFinal_v1.py\", line 158, in <module>\n    stats = bt.run()\n  File \"/opt/anaconda3/envs/tflow/lib/python3.10/site-packages/backtesting/backtesting.py\", line 1170, in run\n    strategy.next()\n  File \"/Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_pp/10_23_2025/backtests_final/T03_FibonacciCrossover_BTFinal_v1.py\", line 114, in next\n    self.buy(size=position_size, sl=sl_price, tp=tp_price)\n  File \"/opt/anaconda3/envs/tflow/lib/python3.10/site-packages/backtesting/backtesting.py\", line 210, in buy\n    return self._broker.new_order(size, limit, stop, sl, tp)\n  File \"/opt/anaconda3/envs/tflow/lib/python3.10/site-packages/backtesting/backtesting.py\", line 713, in new_order\n    raise ValueError(\nValueError: Long orders require: SL (16911.439815447444) < LIMIT (16959.03036) < TP (16952.66036910511)\n\nERROR conda.cli.main_run:execute(124): `conda run python /Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_pp/10_23_2025/backtests_final/T03_FibonacciCrossover_BTFinal_v1.py` failed. (See above for error)\n",
  "execution_time": 1.169424,
  "timestamp": "2025-10-23T10:02:00.652684"
}